tools/clang/test/HLSLFileCheck/hlsl/objects/CbufferLegacy/write_const_debug_info.hlsl

```diff
@@ -0,0 +1,20 @@
+// RUN: %dxc -E main -T ps_6_0 -Gec -Zi %s | FileCheck %s
+
+// Make sure debug info for s.
+// CHECK:!DIGlobalVariable(name: "s.0"
```

Comment:
To make this check meaningful, I think you need to do the thing described in https://github.com/microsoft/DirectXShaderCompiler/blob/master/tools/clang/test/HLSLFileCheck/dxil/debug/_readme.txt:

```c++
  // Exclude quoted source file (see readme)
  // CHECK-LABEL: {{!"[^"]*\\0A[^"]*"}}
```